mgmtsystem/rest_client.py

```diff
@@ -38,7 +39,7 @@ def __init__(self, hostname, auth, protocol="https", port=6443, entry='api/v1',
         else:
             raise RestClientException('Invalid auth object')
 
-    def get(self, entity_type, name=None, namespace=None):
+    def get(self, entity_type, name=None, namespace=None, eval_str=False):
```

Comment:
after putting another thought into it,
a good way to decouple things while allowing flexibility would be to allow a provisional and optional `convert` argument

that way we could call `get(..., convert=eval_strings)` but are also open to do more targeted result validation/type mapping later on